src/providers/apolloClient.tsx

```diff
@@ -124,10 +125,37 @@ export const cache = new InMemoryCache({
   },
 });
 
+const retryLink = new RetryLink({
+  delay: {
+    initial: 800, // Wait 800ms before the first retry after a failed request
+    max: 1200, // Maximum wait time between retries
+    jitter: true,
+  },
+  attempts: {
+    max: 3,
```

Comment:
Docs:
> Starting with initialDelay, the delay of each subsequent retry is increased exponentially, meaning it's multiplied by 2 each time. For example, if initialDelay is 100, additional retries will occur after delays of 200, 400, 800, etc.
> With the jitter option enabled, delays are randomized anywhere between 0ms (instant), and 2x the configured delay. This way you get the same result on average, but with random delays.


I am just wondering if there is a particular logic behind setting the `delay.max` so low?  It's less than 2x the initial, so we don't really get the exponential backoff behavior, if I understand correctly. (And the default is `Infinity`)

Reply:
Okay, sounds good. Thanks. Being kind to our servers seems like a good place to start